crates/parser/veryl.par

```diff
@@ -587,7 +587,7 @@ ModportList: ModportGroup { Comma ModportGroup } [ Comma ];
 
 ModportGroup: { Attribute } ( LBrace ModportList RBrace | ModportItem );
 
-ModportItem: Identifier Colon Direction;
+ModportItem: Identifier Colon (Direction | Import);

```

Comment:
How about adding `Import` into `Direction` instead of this place.
Already `Direction` has `Modport` which is not allowed in this context.
Semantic checker can forbid the usage of `Modport` and `Import` in this context.

Reply:
I merged `Import` with `Direction`.